pom.xml

```diff
@@ -1039,7 +1039,7 @@
                                 <failOnMissing>true</failOnMissing>
                                 <excludedScopes>provided,test</excludedScopes>
                                 <excludedGroups>^(org\.alfresco|com\.alfresco|org\.activiti).*</excludedGroups>
-                                <failIfWarning>true</failIfWarning>
+                                <failOnMissing>true</failOnMissing>
```

Comment:
this tag with the same value is already there

Reply:
removed double tag